### PR TITLE
Competition

### PR DESCRIPTION
**Pull request by e-razdumina**
Add toxic comments classification notebook with ALBERT embeddings and PyTorch Lightning model

- Created a new Jupyter notebook for binary classification of toxic comments
- Implemented a data module with ALBERT text embeddings generation
- Defined a neural network model using PyTorch Lightning with multiple fully connected layers
- Set up training, validation, and test dataloaders with metrics tracking
- Configured model training with AdamW optimizer and binary cross-entropy loss

### CODE REVIEW

**Comment by pacifikus** [comment]
Добрый день!

Уточните, пожалуйста, свой ник на LB соревнования

**Comment by e-razdumina** [comment]
> Добрый день!
> 
> Уточните, пожалуйста, свой ник на LB соревнования

Добрый день! Это я как раз вчера писала: не успела ни дотренить ни инференс сделать, поэтому здесь только код решения

**Comment by pacifikus** [comment]
Ага, поняла! Тогда по коду все хорошо)

**Review by pacifikus — APPROVED** [review]

**Comment by e-razdumina** [comment]
> Ага, поняла! Тогда по коду все хорошо)

Спасибо!
